docs/site/tutorials/todo-list/todo-list-tutorial-controller.md

```diff
@@ -38,6 +38,89 @@ Controller TodoList was created in src/controllers/
 
 And voilà! We now have a set of basic APIs for todo-lists, just like that!
 
+#### Inclusion of Related Models
+
+In order to get our related `Todo`s for each `TodoList`, let's update the
+`schema`.
+
+In `src/models/todo-list.controller.ts`, first import `getModelSchemaRef` from
+`@loopback/rest`.
```

Comment:
FYI: In the future, these changes should be made by `lb4 controller` or `lb4 relation` command automatically, see the following template files:
- [packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs](packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs)
- [packages/cli/generators/relation/templates/controller-relation-template-has-many.ts.ejs](https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/relation/templates/controller-relation-template-has-many.ts.ejs)
- [packages/cli/generators/relation/templates/controller-relation-template-belongs-to.ts.ejs](https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/relation/templates/controller-relation-template-belongs-to.ts.ejs)

Could you please check if we have a story to make those changes? If we don't have it yet, then please create a new one and add it to the epic https://github.com/strongloop/loopback-next/issues/1352 Inclusion of related models

Reply:
Created https://github.com/strongloop/loopback-next/issues/3204